yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher/init.sh

```diff
@@ -44,9 +44,9 @@ cd "$JENKINS_HOME"
 
 # download slave jar
 # TODO some caching mechanism with checksums
-if [[ -x $(command -v wget) ]]; then
+if [[ -x "$(command -v wget)" ]]; then
     wget "${JENKINS_URL}/jnlpJars/slave.jar" -O "slave.jar"
-elif [[ -x $(command -v curl) ]]; then
+elif [[ -x "$(command -v curl)" ]]; then
```

Comment:
`/ # [ -x "" ] && echo OK`
should be the right variant also replace bin bash call in first line to sh